WickedEngine/wiScene.cpp

```diff
@@ -3983,6 +3985,8 @@ namespace wi::scene
 			constexpr uint32_t blocks = envmapRes / GetFormatBlockSize(format);
 			constexpr uint32_t mip_count = GetMipCount(blocks, blocks);
 
+			EnvironmentProbeComponent::supports_MSAA = device->CheckCapability(GraphicsDeviceCapability::SHADER_STORAGE_IMAGE_MULTISAMPLE);
```

Comment:
This is for sure not good here, it's modifying a global from scene.Update() which can be called from any thread.